src/views/GovernanceTaskList/index.scss

```diff
@@ -0,0 +1,88 @@
+.governance-task-list {
+  // Keep heading and order numbers the same
+  &__task-heading,
+  &__item:before {
+    font-size: 1.375em;
+  }
+
+  &__task-list {
+    padding-left: 0;
+    counter-reset: primaryList secondaryList 3;
+
+    &--primary {
+      .governance-task-list__item:before {
```

Comment:
Because we need to style the numbers, it's best to use CSS `:before` to add the numbers rather than the HTML rendered numbers.